src/card.tsx

```diff
@@ -1593,6 +1595,7 @@ export const PosterCard = React.forwardRef<HTMLDivElement, PosterCardProps>(
                                                 </Text2>
                                             </div>
                                         )}
+                                        {extra}
```

Comment:
I guess it is expected to not have any spacings between the extra an the other elements.

Please add some screenshot tests to verify

Reply:
Yes, all `extra` props let this spacing to be the responsibility of the designer

I didn't include screenshots because we don't have screenshots in displayDataCard for extra. But I will do it!

Reply:
added